_episodes/44.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: episode
+title: "44: Seeing clearly with Metabase"
+date: 2023-01-26
+tags: trino metabase visualization client clojure bi reporting
+youtube_id: ""
+wistia_id: ""
+sections:
+   - title: "Release update"
+     desc: "405 and 406"
+     time: 
+---
+
+## Hosts
+
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+
+## Guests
+
+* [Luis Paolini](https://www.linkedin.com/in/luispaolini/), Product Advocate at
+  [Metabase](https://www.metabase.com/)
+* [Andrew DiBiasio](https://www.linkedin.com/in/andrewdibiasio/), Software
+  Engineer at [Starburst](https://starburst.io)
+* [Piotr Leniartek](https://www.linkedin.com/in/piotrleniartek), Project Manager
+  at [Starburst](https://starburst.io)
+
+## Recap of Trino in 2022
+
+Highlights from the blog post [The rabbit reflects on Trino in 2022]({% post_url
+2023-01-10-trino-2022-the-rabbit-reflects %}):
+
+* 
```

Comment:
@colebow we will talk about this together in the show .. can you get a short list of items here?

Reply:
```suggestion
* `MERGE` support
* JSON functions
* Table functions
* Fault-tolerant execution
* Upgrade to Java 17
* New Delta Lake, Hudi, and MariaDB connectors
* Tons and tons of performance improvements
```